### ISSUE TITLE
[BUG] overmind-react with webpack5 has error: `process is not defined` 

### ISSUE THREAD

**Issue by KendraTang** [cerebral/overmind#470]
`overmind-react` is not compatible with webpack v5 since v5 no longer include the polyfill for the Node.js variable ([as described here](https://webpack.js.org/migrate/5/#run-a-single-build-and-follow-advises)), and would cause a console error:

```
Uncaught ReferenceError: process is not defined
```

Here's the dependencies:
```json
{
  "dependencies": {
    "overmind": "^26.0.0",
    "overmind-react": "^27.0.0",
    "react": "^17.0.1",
    "react-dom": "^17.0.1"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/plugin-transform-react-jsx": "^7.12.7",
    "@webpack-cli/serve": "^1.1.0",
    "babel-loader": "^8.2.2",
    "html-webpack-plugin": "^4.5.0",
    "webpack": "^5.9.0",
    "webpack-cli": "^4.2.0",
    "webpack-dev-server": "^3.11.0"
  }
}
```

**Comment by christianalfoni** [comment]
Hm... this is interesting. How are libraries supposed to know what environment they are running in without that? Overmind does a lot of optimisations when built for production and has to know that it is being built for production. Asking on Twitter, and will fix for next release

**Comment by christianalfoni** [comment]
Okay, a suggested fix is now in `next`

**Comment by christianalfoni** [comment]
Released

**Comment by ju1ius** [comment]
Hi @christianalfoni,
I'm still getting this error w/ `overmind-react@28.0.0` : `Uncaught ReferenceError: process is not defined`
This is the offending line: https://github.com/cerebral/overmind/blob/a677abe9223f216dec8342e4edd7e10225d75f68/packages/node_modules/overmind-react/src/index.ts#L20

**Comment by chrisSbarrett** [comment]
Also getting same error with overmind-svelte.

**Comment by chrisSbarrett** [comment]
Solution, as previously discussed is: https://www.npmjs.com/package/rollup-plugin-inject-process-env

**Comment by jdinartejesus** [comment]
I'm still having this issue @christianalfoni with vue-cli, any idea how to quickly fix it?

**Comment by kayfabesoftworks** [comment]
Using:

```
"devDependencies": {
    ...
    "webpack": "^5.37.0",
    "webpack-cli": "^4.7.0",
    "webpack-dev-server": "^3.11.2"
  },
  "dependencies": {
    ...
    "overmind": "^27.0.0",
    "overmind-devtools": "^28.0.0",
    "overmind-react": "^28.0.0",
    ...
  }
```

Still getting this error. I'm current using `patch-package` and replacing these lines:
overmind.js
```
-        const isNode = process && process.title && process.title.includes('node');
+        const isNode = false;
```

overmind-react.js
```
-const isNode = !IS_TEST && process && process.title && process.title.includes('node');
+const isNode = false;
```

**Comment by jdinartejesus** [comment]
@kayfabesoftworks I'm also using this temporary solution during development, but wouldn't be possible when deploying.

Maybe we can create a PR to help @christianalfoni  with a possible solution?

**Comment by eirikhm** [comment]
I fixed this in #495 , but it has not been released yet..

**Comment by vospascal** [comment]
can you plz release this ... cant build my project...  webpack 5 been out for ages also node is not always the title .... can also be 'webpack'
